app/web/src/newhotness/ReviewAttributeItemSourceAndValue.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div
+    v-if="displayKind !== 'hidden'"
+    :class="
+      clsx(
+        'flex flex-row items-center gap-xs font-mono h-10 p-xs',
+        old
+          ? themeClasses('text-neutral-600', 'text-neutral-400')
+          : themeClasses('bg-success-200', 'bg-success-900'),
+      )
+    "
+  >
+    <div
+      :class="
+        clsx(
+          'text-xl flex-none w-sm text-center',
+          !old && themeClasses('text-success-600', 'text-success-500'),
+        )
+      "
+    >
+      {{ old ? "-" : "+" }}
+    </div>
+    <!-- <div class="text-2xs">
+      {{ rawValue }} / {{ rawSource }}
+    </div> -->
+    <TruncateWithTooltip
+      v-if="displayKind === 'value' || displayKind === 'complex'"
+      :class="clsx('py-2xs', old && 'line-through')"
```

Comment:
Neat! Good way to do the strikethrough. I'll probably need it for component history